### PR TITLE
Update HT,  LCT2 and LC modifier libraries

### PR DESCRIPTION
**Pull request by Legendsmith**
Updated the data files the latest update, also make them use the right modifier stages thanks to Fantell. UT is still to c̨om̧͝ȩ́. Ḩ̵̜͈̻̀E̴̬̗̺̠̱̰̗ ̩͕̘̞͕̣͎̤̦C̵̴̝͕̖Ǫ̧̗̣̼M҉̭̪̜̖̯̤͖͚̤Ȩ̫͘S̶͇̤͉̫̼͝. ``I dream`` in Xͬ̏̃̚M͛͂͗̏͒ͬ͊L͊ͯͪͬ now.

### CODE REVIEW

**Comment by Legendsmith** [comment]
UT added. After talking with Fantell I've made two versions of the modifiers within UT and HT, because RAW they don't use CF, but it's a common rule now (Confirmed by Kromm) to have them use CF.

**Comment by richardwilkes** [comment]
Looks like there are some conflicts with previous PRs

**Comment by Legendsmith** [comment]
Resolved the problem.

File: Library/High Tech/High Tech Equipment Modifiers.eqm [hunk 1]
```diff
@@ -1,32 +1,12 @@
 <?xml version="1.0" encoding="US-ASCII" ?>
 <eqp_modifier_list id="ac9278ab-975d-459a-9eab-a6cfd2f1ee27" version="1">
+	<eqp_modifier_container version="1" open="no" enabled="no">
+		<name>Note</name>
+		<notes>Both Cost Factor and Multiplicative Modifiers for equipment are provided here. Kromm reccomends using Cost Factor which was introduced in Low-Tech, rather than the multiplicative cost modifiers. But some GMs may wish to use multiplicative.</notes>
```
Comment (richardwilkes) [inline Library/High Tech/High Tech Equipment Modifiers.eqm:5]: recommend is mis-spelled

File: Library/High Tech/High Tech Equipment Modifiers.eqm [hunk 2]
```diff
@@ -87,298 +67,657 @@
...
+			<name>SM -4</name>
+			<reference>HT10</reference>
+			<cost cost_type="to final cost">x0.05</cost>
+			<weight weight_type="to final base weight">x1/20</weight>
+		</eqp_modifier>
 	</eqp_modifier_container>
 	<eqp_modifier_container version="1" open="yes" enabled="no">
 		<name>Equipment Modifiers</name>
 		<reference>HT10</reference>
+		<notes>Cost Factor Coversion as per LT14</notes>
```
Comment (richardwilkes) [inline Library/High Tech/High Tech Equipment Modifiers.eqm:98]: Conversion is mis-spelled

File: Library/High Tech/High Tech Equipment Modifiers.eqm [hunk 3]
```diff
@@ -87,298 +67,657 @@
...
-			<cost cost_type="to final cost">x2</cost>
-			<weight weight_type="to final base weight">x2/3</weight>
+			<cost cost_type="to base cost">+9 CF</cost>
+			<notes>+3 Reaction</notes>
 		</eqp_modifier>
 	</eqp_modifier_container>
 	<eqp_modifier_container version="1" open="yes" enabled="no">
 		<name>Ammunition and Warhead Modifiers</name>
+		<reference>HT162</reference>
+		<notes>Cost Factor Coversion as per LT14</notes>
```
Comment (richardwilkes) [inline Library/High Tech/High Tech Equipment Modifiers.eqm:145]: Conversion again

File: Library/Ultra Tech/Ultra Tech Equipment Modifiers.eqm [hunk 4]
```diff
@@ -1,62 +1,12 @@
...
-	</eqp_modifier>
-	<eqp_modifier version="1">
-		<name>Expensive</name>
-		<reference>UT15</reference>
-		<cost cost_type="to final cost">x2</cost>
-		<weight weight_type="to final base weight">x2/3</weight>
-	</eqp_modifier>
+	<eqp_modifier_container version="1" open="no" enabled="no">
+		<name>Note</name>
+		<notes>Both Cost Factor and Multiplicative Modifiers for equipment are provided here. Kromm reccomends using Cost Factor which was introduced in Low-Tech, rather than the multiplicative cost modifiers. But some GMs may wish to use multiplicative.</notes>
```
Comment (richardwilkes) [inline Library/Ultra Tech/Ultra Tech Equipment Modifiers.eqm:5]: Recommends again

File: Library/Ultra Tech/Ultra Tech Equipment Modifiers.eqm [hunk 5]
```diff
@@ -321,111 +566,225 @@
...
+			<name>Armor Piercing Hyperdense Dart</name>
 			<reference>UT156</reference>
-			<cost cost_type="to final cost">x10</cost>
+			<cost cost_type="to final base cost">x10</cost>
 		</eqp_modifier>
 		<eqp_modifier version="1">
-			<name>EMP</name>
-			<reference>UT157</reference>
-			<cost cost_type="to final cost">x10</cost>
+			<name>Armor-Piercing Enhanced Penertrator</name>
```
Comment (richardwilkes) [inline Library/Ultra Tech/Ultra Tech Equipment Modifiers.eqm:574]: I believe that should be Penetrator

File: Library/Ultra Tech/Ultra Tech Equipment Modifiers.eqm [hunk 6]
```diff
@@ -321,111 +566,225 @@
...
+			<reference>UT154</reference>
+			<cost cost_type="to final base cost">x1</cost>
+		</eqp_modifier>
+		<eqp_modifier version="1">
+			<name>High Explosive Multi-Purpose</name>
+			<reference>UT155</reference>
+			<cost cost_type="to final base cost">x2</cost>
+		</eqp_modifier>
+		<eqp_modifier version="1">
+			<name>Holllow-Point</name>
```
Comment (richardwilkes) [inline Library/Ultra Tech/Ultra Tech Equipment Modifiers.eqm:641]: Hollow is misspelled

**Comment by Legendsmith** [comment]
Typos fixed.

**Pull request merged**